pkg/guacanalytics/patchPlanning.go

```diff
@@ -132,9 +144,7 @@ func SearchDependenciesFromStartNode(ctx context.Context, gqlClient graphql.Clie
 		q.nowNode.Expanded = true
 		q.nodeMap[q.now] = q.nowNode
 	}
-
-	return q.nodeMap, nil
-
+	return nil, nil
```

Comment:
why do you return nil for the nodeMap?

Reply:
It is retuning `nil` because `q` is being mutated and we return `q` at the end of `SearchDependenciesFromStartNode`.

I just push a small change removing the first return value of `map[string]BfsNode`. The function `bfsOfDependencies` now only returns an error because if there is an error, `SearchDependenciesFromStartNode` will return `nil` and the error, and if there isn’t an error it will return the `q.nodeMap` and `nil`.